script.js

```diff
@@ -1,11 +1,20 @@
+/* eslint-disable no-unused-vars */
+/* eslint-disable no-alert */
+/* eslint-disable no-useless-escape */
+/* eslint-disable prefer-destructuring */
+/* eslint-disable no-plusplus */
 /* eslint-disable func-names */
 /* eslint-disable linebreak-style */
 const hamburger = document.querySelector('.hamburger');
 const ulCont = document.querySelector('.ul-cont');
-// const modal = document.querySelector('.popup-window');
-// const overlay = document.querySelector('.overlay');
-// const openModalBtn = document.querySelector('.card-button');
-// const closeModalBtn = document.querySelector('.closeIcon-popup');
+const modal = document.querySelector('.popup-window');
+const overlay = document.querySelector('.overlay');
+const openModalBtn = document.querySelector('.card-button');
+const openModalBtn2 = document.querySelector('.card-button2');
+const openModalBtn3 = document.querySelector('.card-button3');
+const openModalBtn4 = document.querySelector('.card-button4');
+
+const closeModalBtn = document.querySelector('.closeIcon-popup');
```

Comment:
- [ ] As seen in the screenshot below, the Close button does not diplay in the desktop version and as such one cannot close your pop up. Clicking the X button to close the pop up is a project requirement. Kindly fix this issue.
![image](https://user-images.githubusercontent.com/62849872/210359389-46837de7-4e6e-455c-93cd-4a3d4d8b26b8.png)

Details about the project are overlaying to the left and not displaying . Check the screenshot above and also fix that.
- Your design
![image](https://user-images.githubusercontent.com/62849872/210359807-59dd5f58-a362-461f-95a7-3900863d43bd.png)
- Figma design
![image](https://user-images.githubusercontent.com/62849872/210359954-6027a5dd-043d-46fa-9208-b42f3704974e.png)

Kindly make your desktop modal pop up to look exactly like the figma design as shown in the screenshot above.